docs/src/best-practices-js.md

```diff
@@ -80,15 +110,24 @@ Playwright has a [test generator](./codegen.md) that can generate tests and pick
 
 #### Use `codegen` to generate locators
 
-To pick a locator you can run the `codegen` command and click on the pick locator button. Then hover over your page in the browser window and click on the element you want to pick. You can then copy and paste this locator into your code. You can also use `codegen` to record a test for you.
+To pick a locator run the `codegen` command followed by the URL you would like to pick a locator from.
```

Comment:
COULD:
```suggestion
To pick a locator run the `codegen` command followed by the URL that you would like to pick a locator from.
```

adding this pronoun might be more appropriate, because the pronoun "that" is referring back to the noun "URL" which is the antecedent. It's more clear and specific.